### PR TITLE
Distance

### PR DESCRIPTION
**Pull request by devanp92**
Adding distance + test

### CODE REVIEW

**Comment by devanp92** [comment]
I don't know why the Catalan numbers are being adding since I created a new branch called distance and pushed only the distance stuff to it.

File: lib/function/geometry/distance.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+ function _2d(x) {
+  return x.length === 2 && typeof x[0] === 'number' && typeof x[1] === 'number';
+}
+
+function _distance2d(x1, y1, x2, y2){
+  var radicant = Math.pow(y2-y1, 2) + Math.pow(x2-x1, 2);
+  var result = Math.pow(radicant, 0.5);
+  return result;
+}
```
Comment (Yaffle) [inline lib/function/geometry/distance.js]: Math.hypot ?

Comment (devanp92) [inline lib/function/geometry/distance.js]: Good point. Didn't think about that.

**Comment by josdejong** [comment]
Thanks @devanp92. And sorry for the late reply. This looks like a good start (2d points). Later we can extend with support for 3D points. 

Can you please redo the PR without the `catalan` functions (these are merged already to `develop`). Thanks!

**Comment by devanp92** [comment]
@josdejong Travis-CI is failing because of the
    
    Math.hypot()

[Math.hypot](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/hypot) was added to ES6. Is math.js not ES6 compatible?

**Comment by josdejong** [comment]
math.js indeed still misses new functions added in ES6, see #235.

**Comment by josdejong** [comment]
@devanp92 I see this PR is still open, can you redo the PR without the catalan functions? These are merged already to develop? Thanks.

**Comment by Yaffle** [comment]
https://github.com/paulmillr/es6-shim/blob/master/es6-shim.js#L1619

**Comment by josdejong** [comment]
Thanks for the tip @Yaffle :) I added links to these examples and example code on MDN to #235

**Pull request closed without merging**